front/src/app/core/library/GeneralCue.ts

```diff
@@ -0,0 +1,17 @@
+import { ICue } from '../interfaces/ICue';
+import { CueTrigger } from '../enumerations/CueTrigger';
+import { ICueble } from '../interfaces/ICueble';
+import { IFade } from '../interfaces/IFade';
+
+export class GeneralCue implements ICue{
+    faderValue: any;    trigger: CueTrigger[];
+    triggerImmediate: number;
+    await: boolean;
+    cueList;
```

Comment:
Again a type seems to be missing for cueList.
Also: why is trigger on the same line as faderValue?